src/content/en/tools/chrome-devtools/profile/rendering-tools/analyze-runtime.markdown

```diff
@@ -1,82 +1,73 @@
   javascript:
     -  
       title: "Optimize JavaScript Execution"
       href: fundamentals/performance/rendering/optimize-javascript-execution
-      section:
-        title: "Optimize JavaScript Execution"
-        href: fundamentals/performance/rendering/optimize-javascript-execution
 translation_priority: 0
 key-takeaways:
-  layout-issues:
-    - "Don't write JavaScript that forces the browser to recalculate layout. Separate read and write functions, and perform reads first."
-    - "Don't over-complicate your CSS. Use less CSS and keep your CSS selectors simple."
-    - "Avoid layout as much as possible. Choose CSS that doesn't trigger layout at all."
-    - "Painting can take up more time than any other rendering activity. Watch out for paint bottlenecks."
-
+  - "Don't write JavaScript that forces the browser to recalculate layout. Separate read and write functions, and perform reads first."
+  - "Don't over-complicate your CSS. Use less CSS and keep your CSS selectors simple."
+  - "Avoid layout as much as possible. Choose CSS that doesn't trigger layout at all."
+  - "Painting can take up more time than any other rendering activity. Watch out for paint bottlenecks."
 notes:
   css-triggers:
     - "Learn how Paul Lewis built <a href='https://aerotwist.com/blog/css-triggers'>CSS Triggers</a>."
 ---
 
-<p class="intro">
-  Users expect pages to be interactive and smooth. Each stage in the pixel pipeline represents an opportunity to introduce 'jank'. Learn about the tools and strategies to identify and fix common problems that slow down runtime performance.
-</p>
+<p class="intro">Users expect pages to be interactive and smooth. Each stage 
+in the pixel pipeline represents an opportunity to introduce jank. Learn about 
+tools and strategies to identify and fix common problems that slow down 
+runtime performance.</p>
 
 {% include shared/toc.liquid %}
 
-{% include shared/takeaway.liquid list=page.key-takeaways.layout-issues %}
+{% include shared/takeaway.liquid list=page.key-takeaways %}
 
-## How to identify JavaScript bottlenecks
+## JavaScript 
 
-JavaScript calculations, especially ones
-that trigger extensive visual changes,
-can stall application performance.
-Don't let badly-timed or long-running JavaScript interfere
-with users interacting with your site.
+JavaScript calculations, especially ones that trigger extensive visual changes,
+can stall application performance. Don't let badly-timed or long-running 
+JavaScript interfere with user interactions.
 
 ### Tools
 
-The simplest way to test JavaScript performance is to view Scripting events
-in the [Chrome DevTools Timeline](/web/tools/chrome-devtools/profile/evaluate-performance/timeline-tool).
-If any of the scripting events seem suspiciously long,
-enable the JS Stacks at the top of the Timeline tool:
-
-![JavaScript Profiler](imgs/js-stack.png)
+Make a **Timeline** [recording][recording] and look for suspiciously long 
```

Comment:
This is a reference-style link. See bottom of this section: https://daringfireball.net/projects/markdown/syntax#link